src/pages/integrations/index.tsx

```diff
@@ -0,0 +1,279 @@
+    </a>
+  );
+};
+
+interface FilterBarProps {
+  categoryFilter: string;
+  setCategoryFilter: (f: string) => void;
+  searchQuery: string;
+  setSearchQuery: (q: string) => void;
+}
+
+function FilterBar({
+  categoryFilter,
+  setCategoryFilter,
+  searchQuery,
+  setSearchQuery,
+}: FilterBarProps) {
+  return (
+    <div className="container margin-vert--md">
+      <div className={clsx("padding-top--md", styles.filter)}>
+        <div
+          className={clsx(
+            "navbar__search margin-right--md",
+            styles.filter__search
+          )}
+        >
+          <input
+            className="navbar__search-input"
+            placeholder="Search integrations"
+            onInput={(e) => {
+              setSearchQuery((e.target as HTMLInputElement).value);
+            }}
+          />
+        </div>
+        <div
+          className={clsx(
+            "dropdown dropdown--hoverable dropdown--right",
+            styles.filter__dropdown
+          )}
+        >
+          <button className="button button--primary padding-horiz--sm">
+            <IconFilter />
+            {categoryFilter || "All"}
+          </button>
+          <ul className="dropdown__menu">
+            <li>
+              <a
+                className="dropdown__link"
+                onClick={() => setCategoryFilter("")}
+              >
+                All
+              </a>
+            </li>
+            {Array.from(TAGS).map((tag: Tag) => (
+              <li key={tag}>
+                <a
+                  className="dropdown__link"
+                  onClick={() => {
+                    setCategoryFilter(tag);
+                  }}
+                >
+                  {tag}
+                </a>
+              </li>
+            ))}
+            <li>
+              <a
+                className="dropdown__link"
+                onClick={() => setCategoryFilter("v1")}
+              >
+                v1
+              </a>
+            </li>
+          </ul>
+        </div>
+        <ul className="pills margin--none">
+          <li
+            className={clsx(
+              "pills__item",
+              categoryFilter || "pills__item--active"
+            )}
+            value={searchQuery}
+            onClick={() => setCategoryFilter("")}
+          >
+            All
+          </li>
+          {Array.from(TAGS).map((tag: Tag) => (
+            <li
+              key={tag}
+              onClick={() => {
+                setCategoryFilter(tag);
+              }}
+              className={clsx("pills__item", {
+                "pills__item--active": categoryFilter === tag,
+              })}
+            >
+              {tag}
+            </li>
+          ))}
+          <li
+            className={clsx("pills__item", {
+              "pills__item--active": categoryFilter === "v1",
+            })}
+            value={searchQuery}
+            onClick={() => setCategoryFilter("v1")}
+          >
+            v1
+          </li>
```

Comment:
To make it clear that you are switching between `v1` and `v2` integrations, maybe make this a dropdown?